packages/itwin/grouping-mapping-widget/src/widget/components/GroupAction.tsx

```diff
@@ -58,14 +75,18 @@ const GroupAction = ({
   const [validator, showValidationMessage] = useValidator();
   const [isLoading, setIsLoading] = useState<boolean>(false);
   const [isRendering, setIsRendering] = useState<boolean>(false);
-  const [currentPropertyList, setCurrentPropertyList] = React.useState<PropertyRecord[]>([]);
+  const [currentPropertyList, setCurrentPropertyList] = React.useState<
```

Comment:
Inconsistent use of `useState`. Everywhere else above you do not use the `React.` prefix.